src/magic_folder/config.py

```diff
@@ -1443,9 +1464,31 @@ def create_magic_folder(self, name, magic_path, author,
         :param unicode upload_dircap: the write-capability of the
             directory we upload data into.
 
+        :param int poll_interval: how often to scan for remote changes
+            (in seconds).
+
+        :param Optional[int] scan_interval: how often to scan for local changes
+            (in seconds).
+
         :returns: a MagicFolderConfig instance
         """
         valid_magic_folder_name(name)
+        if not isinstance(poll_interval, int) or poll_interval <= 0:
+            raise APIError(
+                code=http.BAD_REQUEST,
+                reason="Poll interval must be a positive integer (not '{!r}').".format(
+                    poll_interval
+                ),
+            )
+        if scan_interval is not None and (
+            not isinstance(scan_interval, int) or scan_interval <= 0
```

Comment:
[In the DB above, 0 is an allowed value](https://github.com/LeastAuthority/magic-folder/pull/468/files#diff-db422d0a72330e38baed0b613e6f69f9f154b6e67e0db758516627d0492fb634R147), but here 0 is forbidden.

Reply:
I've added a constraint to the DB.